package.json

```diff
@@ -30,6 +30,7 @@
     "gsap": "^2.1.2",
     "hammerjs": "^2.0.8",
     "instafeed": "^2.0.0",
+    "instafeed.js": "^1.4.1",
```

Comment:
this was the core of the issue. instafeed either died or changed names, this is, for all purposes, the same package